test/specs/runApexTests.e2e.ts

```diff
@@ -146,32 +225,47 @@ describe('Run Apex Tests', async () => {
     expect(apexTestsSection.elem).toBePresent();
 
     // Open an existing apex test and modify it
-    const textEditor = await editorView.openEditor('ExampleApexClass1Test.cls') as TextEditor;
+    const textEditor = (await editorView.openEditor(
+      'ExampleApexClass1Test.cls'
+    )) as TextEditor;
     const testText = [
       '@IsTest',
       'public class ExampleApexClass1Test {',
       '\t@IsTest',
       '\tstatic void validateSayHello() {',
-      '\t\tSystem.debug(\'Starting validate\');',
-      '\t\tExampleApexClass1.SayHello(\'Andres\');',
-      '\t\tSystem.assertEquals(1, 1, \'all good\');',
+      "\t\tSystem.debug('Starting validate');",
```

Comment:
I like the other way, with single quotes, and then escaping the inner quotes better:
```
const testText = [
      '@IsTest',
      'public class ExampleApexClass1Test {',
      '\t@IsTest',
      '\tstatic void validateSayHello() {',
      '\t\tSystem.debug(\'Starting validate\');',
      '\t\tExampleApexClass1.SayHello(\'Andres\');',
      '\t\tSystem.assertEquals(1, 1, \'all good\');',
      '\t}',
      '}'
    ].join('\n');
```

Reply:
Uh good catch, I can't remember having changed this on purpose, I think it was the formatter. I'll change this to be the same way as in apexUtils